.vscode/launch.json

```diff
@@ -15,7 +15,7 @@
         "--server",
         "--migrations"
       ],
-      "console": "integratedTerminal"
+      // "console": "integratedTerminal"
```

Comment:
To remove

Reply:
yea, I'm still having issues with my VSCode randomly... it works when I first start it, then starts failing randomly after a few script restarts